docs/root/intro/version_history.rst

```diff
@@ -5,6 +5,8 @@ Version history
 ================
 * access log: added a new field for response code details in :ref:`file access logger<config_access_log_format_response_code_details>` and :ref:`gRPC access logger<envoy_api_field_data.accesslog.v2.HTTPResponseProperties.response_code_details>`.
 * dubbo_proxy: support the :ref:`Dubbo proxy filter <config_network_filters_dubbo_proxy>`.
+* ext_authz: added option to `ext_authz` that allows the filter clearing route cache.
```

Comment:
Sorry, this one was added from previous PR right?

Reply:
Yes, I think I've just swapped places. Not sure.